tests/test-cases.js

```diff
@@ -1046,6 +1055,36 @@ var PARSE_TESTS = [
 			errors: []
 		}
 	},
+	{
+		description: "Null lines remain unaffected by 'skip empty lines' config",
+		notes: "",
+		input: 'a,b\nc,d\n\n,\n',
+		config: { skipEmptyLines: true },
+		expected: {
+			data: [['a', 'b'], ['c', 'd'], ["", ""]],
+			errors: []
+		}
+	},
+	{
+		description: "Skip null lines with only empty values",
+		notes: "",
+		input: 'a,b\nc,d\n,\n,\n',
+		config: { skipNullLines: true },
+		expected: {
+			data: [['a', 'b'], ['c', 'd']],
+			errors: []
+		}
+	},
+	{
+		description: "Skip null lines and empty lines work well with each other",
+		notes: "",
+		input: 'a,b\n,\n\nc,d',
+		config: { skipNullLines: true, skipEmptyLines: true },
+		expected: {
+			data: [['a', 'b'], ['c', 'd']],
+			errors: []
+		}
```

Comment:
Can we add a test to ensure that empty lines are preserved when skipEmptyLines is false but skipNullLlines is true? 

It will be great to test the skipNullLines combined with header option.

Reply:
Ah. That test was vital! `skipNullLines` is frustratingly also doing the job of `skipEmptyLines`. I'm figuring out a way to differentiate them now.

Reply:
Unfortunately, this might not be easy to fix. By the time the data gets to `isEmptyLine`, for an emptyLine and a emptyFieldsLine the data structure looks the same.

For now, my company may just have to stick with the forked version of this library (or live with a few spreadsheets looking odd). I'll add the failing test I have, in case you've any ideas!

Reply:
Not sure if there is a use case for that, just wanted to see what's the behaviour of both flags together. Probably we can keep this behaviour but we should add a warning on the docs that explains that skipNullLines implies also skipEmptyLines